website/docs/docs/use-dbt-semantic-layer/setup-sl.md

```diff
@@ -0,0 +1,106 @@
+
+<SlSetUp/>
+
+<!--
+1. Create a new environment in dbt Cloud by selecting **Deploy** and then **Environments**.
+2. Select **dbt Version 1.6** (or the latest) and enter your deployment credentials.
+3. To configure the new Semantic Layer, you must have a successful run in your new environment. We recommend running `dbt ls` since `dbt build` won’t succeed until you’ve created and defined semantic models and metrics.
+4. To enable the dbt Semantic Layer, go to the **Account Settings** page and then select the specific project you want to enable the Semantic Layer for.
+5. In the **Project Details** page, select **Configure Semantic Layer.** This will prompt you to enter data platform connection credentials for the Semantic Layer and select the environment where you want to enable the Semantic Layer. We recommend using a less privileged set of credentials when setting up your connection. The semantic layer requires SELECT and CREATE TABLE permissions.
+6. After you’ve entered your credentials, you should see connection information that will allow you to connect to downstream tools. If the tool you are using can connect with JDBC, you can save the **JDBC URL** or each of the individual components provided (e.g., environment id, host). Alternatively, if the tool you connect to uses the Semantic Layer GraphQL API, save the GraphQL API host information.
+7. Next, go back to the **Project Details** page and select **Generate Service Token** to create a Semantic Layer service token. Save this token for later.
+8. You’re done 🎉! The semantic layer should is now enabled for your project. 
+-->
+
+</VersionBlock>
+
+<VersionBlock lastVersion="1.5">
+
+import LegacyInfo from '/snippets/_legacy-sl-callout.md';
+
+<LegacyInfo />
+
+With the dbt Semantic Layer, you'll be able to centrally define business metrics, reduce code duplication and inconsistency, create self-service in downstream tools, and more. Configure the dbt Semantic Layer in dbt Cloud to connect with your integrated partner tool. 
+
+## Prerequisites
+
+Before you set up the dbt Semantic Layer, make sure you meet the following:
+
+<Snippet path="sl-prerequisites" />
+
+## Set up dbt Semantic Layer
+
+:::tip
+If you're using the legacy Semantic Layer, we **highly** recommend you [upgrade your dbt version](/docs/dbt-versions/upgrade-core-in-cloud) to dbt v1.6 or higher to use the new dbt Semantic Layer. Refer to the dedicated [migration guide](/guides/migration/sl-migration) for more info.
+
+:::
+
+Before you continue with the following steps, you **must** have a multi-tenant dbt Cloud account hosted in North America. 
+ * Team and Enterprise accounts can set up the Semantic Layer and [Discovery API](/docs/dbt-cloud-apis/discovery-api) in the integrated partner tool to import metric definitions. 
+ * Developer accounts can query the Proxy Server using SQL but won't be able to browse dbt metrics in external tools, which requires access to the Discovery API.
```

Comment:
great info in the 2 bullets above. instead of a list, sugg adding an intro paragraph (which is common practice before a set of steps) and work this info into it